.gitignore

```diff
@@ -9,3 +9,4 @@ test.db
 .sass-cache
 geekup/static/img/uploads
 error.log
+testing.py
```

Comment:
`settings.py`, `development.py` and `testing.py` should be prefixed with `instance/` so that other files that share the name aren't accidentally excluded.